### ISSUE TITLE
Hide types of ammo behind checkpoints

### ISSUE THREAD

**Issue by VRabadan** [dcs-retribution/dcs-retribution#187]
### Is your feature request related to a problem? Please describe.

This os not related to any existing problem

### Describe the solution you'd like

I thought it to be cool that types of ammo to be "blocked" until the player has conquered certain checkpoints and the ammo depots containing that ammo.

Ammo depots should also be not destroyed. This would make more sense after we can repair strike targets; so we can "take out" from the enemy the hability to use certain ammo types but then get it back after taking the checkpoint.

In short:
Ammo Depot - If no ammotype is specified; act as default (like it works now)
If ammo type/s are specified; that is what that specific depot holds and that is what the associated checkpoint is allowed to use.
If the depot is destroyed, the ammo goes along with it unless it's repaired first.

It would be ideal that this would also work for roadbases and FARPs.

**Comment by holyorangejuice** [comment]
A warehouse system in retribution UI would be a great addition. It would function like restrict weapons by date. It's purely UI based. Doesn't integrate with DCS (limitations..ty ED) and so players can override by rearming at a base.

At least it would be something.

**Comment by Raffson** [comment]
Well, I think we should probably look into how much work it would take to actually link it all up with DCS 😬
